src/Package/Impl/ExportDialog/ExportPDFDialog.xaml.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Windows.Controls;
+using Microsoft.Common.Core;
+using Microsoft.VisualStudio.R.Package.Wpf;
+
+namespace Microsoft.VisualStudio.R.Package.ExportDialog {
+    public partial class ExportPDFDialog : PlatformDialogWindow, INotifyPropertyChanged {
+
+        private const double MIN = 1.0;
+        private const double MAX = 1100.0;
+        private double _prevValidWidth = MIN;
+        private double _prevValidHeight = MIN;
+        private bool _viewPlot;
+        private ExportArguments _pdfArguments;
+        private PDFExportOptions _pdfPaperType;
+        private List<PDFExportOptions> _pdfPapers;
+        private List<OrientationEnum> _orientations;
+        private List<PdfPrintOptionsEnum> _printOptions;
+        public event PropertyChangedEventHandler PropertyChanged;
+
+        public List<PDFExportOptions> PDFPapers => _pdfPapers;
+        public List<OrientationEnum> Orientations => _orientations;
+        public List<PdfPrintOptionsEnum> PrintOptions => _printOptions;
+
+        public PDFExportOptions SelectedPDFPaperType {
+            get { return _pdfPaperType; }
+            set {
+                _pdfPaperType = value;
+                OnPropertyChanged("SelectedPDFPaperType");
```

Comment:
You can use `nameof(SelectedPDFPaperType)` for these. It's better because refactoring rename operations will automatically update it.  Same for others below.